.gitlab-ci.yml

```diff
@@ -73,23 +72,30 @@ check-wasm:
         cargo check --verbose --no-default-features --target wasm32-unknown-unknown --manifest-path ${crate}/Cargo.toml;
       done
 
+build-cli:
+  stage:                           check
+  <<:                              *docker-env
+  script:
+    - cargo build --manifest-path cli/Cargo.toml --release
+    - ./target/release/cargo-contract -V
+  artifacts:
+    name:                          cli_${CI_COMMIT_SHORT_SHA}
+    expire_in:                     24 hrs
+    paths:
+      - ./target/release/cargo-contract
+
+
 #### stage:                        workspace
 
 build-std:
   stage:                           workspace
   <<:                              *docker-env
-  dependencies:
-    - check-std
   script:
-    - for crate in ${ALL_CRATES}; do
-        cargo build --verbose --all-features --release;
-      done
+    - cargo build --verbose --all-features --release
```

Comment:
Missing the `--all` flag if you want to go without the loop ... however, I really do not recommend doing so. Why have you changed it back?